BookingsApi/BookingsApi/Controllers/V1/HearingParticipantsController.cs

```diff
@@ -228,7 +228,7 @@ public async Task<IActionResult> UpdateHearingParticipants(Guid hearingId, [From
                 return ValidationProblem(ModelState);
             }
 
-            var hearing = await _hearingParticipantService.UpdateParticipants(request, videoHearing);
+            var hearing = await _hearingParticipantService.UpdateParticipants(request, videoHearing, true);
```

Comment:
as this is an optional argument, please could you specify its name:

`var hearing = await _hearingParticipantService.UpdateParticipants(request, videoHearing, sendNotification: true);`

Reply:
done